docs/reference/modules/node.asciidoc

```diff
@@ -340,6 +340,9 @@ xpack.ml.enabled: true <1>
 ----
 <1> The `xpack.ml.enabled` setting is enabled by default.
 
+If you want the node to use {ccs}, add `remote_cluster_client` to the list of
+roles. See <<remote-node>>.
+
```

Comment:
```suggestion

If you are using {ccs} in any way it is strongly recommended to allow {ml} nodes to do remote searches unless you have a very good reason not to. It will likely lead to confusion and frustration if cross cluster searches usually work perfectly but fail when used in {ml} job or {dfeed} configs.
```